app/models/hubstats/pull_request.rb

```diff
@@ -6,10 +6,13 @@ class PullRequest < ActiveRecord::Base
     scope :updated_in_date_range, lambda {|start_date, end_date| where("hubstats_pull_requests.updated_at BETWEEN ? AND ?", start_date, end_date)}
     scope :created_in_date_range, lambda {|start_date, end_date| where("hubstats_pull_requests.created_at BETWEEN ? AND ?", start_date, end_date)}
     scope :merged_in_date_range, lambda {|start_date, end_date| where("hubstats_pull_requests.merged").where("hubstats_pull_requests.merged_at BETWEEN ? AND ?", start_date, end_date)}
+    scope :created_in_past_year, lambda {|date| where("hubstats_pull_requests.created_at > ?", date)}
```

Comment:
I don't see anything in this scope that limits it to the past year.  You should update the logic to or update the name to be more accurate.